packages/reports/src/Stores/Modules/Profit/profit-store.js

```diff
@@ -108,16 +108,16 @@ export default class ProfitTableStore extends BaseStore {
         this.is_loading = false;
     }
 
-    fetchOnScroll = debounce(left => {
+    fetchOnScroll = debounce((left, shouldFiltrateContractTypes) => {
```

Comment:
```suggestion
    fetchOnScroll = debounce((left, shouldFilterContractTypes) => {
```
same for `async fetchNextBatch(shouldFilterContractTypes)` :)